example-project/build.gradle

```diff
@@ -89,7 +89,7 @@ protobuf {
     plugins {
         grpc { artifact = "io.grpc:protoc-gen-grpc-java:${versions.grpc}" }
         kroto {
-            artifact = "com.github.marcoferrer.krotoplus:protoc-gen-kroto-plus:${versions.krotoplus}:jvm8@jar"
+            path = "${rootDir}/../protoc-gen-kroto-plus/build/libs/protoc-gen-kroto-plus-${versions.krotoplus}-${osdetector.classifier}.exe"
```

Comment:
was this in error? The example project should still be referencing the remote dependency, shouldn't it?

Reply:
removing the jvm8@jar part should do it

Reply:
Good catch. It was to test out the windows support locally.